frontend/src/metabase/dashboard/containers/AutomaticDashboardApp.jsx

```diff
@@ -94,7 +94,7 @@ class AutomaticDashboardApp extends React.Component {
               <Icon name="bolt" className="text-gold mr1" size={24} />
               <h2>{dashboard && dashboard.name}</h2>
               <ActionButton
-                className="ml-auto bg-green text-white"
+                className="ml-auto Button--success bg-green-saturated-hover"
```

Comment:
I'm thinking we should probably just fix the hover state on `Button--success` if we're not a fan of the color rather than monkey patch this particular button.

Reply:
Only reason I hadn't gone that way is because success buttons are currently used in a disabled success state like when sending a pulse test, and I don't think we'd want to have a hover state in that situation. But maybe I'll just make a separate green button class.

Reply:
Gotcha. I think the way to properly do this is change the default hover to what you'd like it to be and then we can go in and make sure it's not applied in that special case. I can help with that if you'd like.